assets/app/lib/settings.rb

```diff
@@ -34,6 +35,7 @@ module Settings
 
     def self.included(base)
       base.needs :user, default: nil, store: true
+      base.send :include, Lib::Color
```

Comment:
@jenf this should prevent further mishaps and also alleviate some require/include load.